libr/core/fortune.c

```diff
@@ -94,16 +94,27 @@ R_API void r_core_fortune_list(RCore *core) {
 }
 
 static char *getrandomline(RCore *core) {
-	const char *types = (char *)r_config_get (core->config, "cfg.fortunes.type");
+	RList *types = r_str_split_duplist (
+		(char *)r_config_get (core->config, "cfg.fortunes.type"), ",", false);
+	if (!r_list_length (types)) {
```

Comment:
maybe makes sense to have an alias named 'all' to select all fortunefiles (maybe in another pr)

Reply:
should be * imo

Reply:
yeah, * sounds good. no conflicts with flienames